.github/workflows/ci.yml

```diff
@@ -98,6 +125,20 @@ jobs:
           echo "::set-env name=CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER::/usr/bin/aarch64-linux-gnu-gcc-5"
           echo "::set-env name=QEMU_LD_PREFIX::/usr/aarch64-linux-gnu"
 
+      - name: Install rust-embedded/cross
+        # see https://github.com/rust-embedded/cross
+        if: contains(matrix.config.target, 'android')
+        run: |
+          cargo install cross
+          echo 'alias cargo=cross' >> $HOME/.bash_profile
+
+      - name: Build android docker images
+        if: contains(matrix.config.target, 'android')
+        run: |
+          sudo apt update
+          sudo apt install docker
+          docker build --target ${{ matrix.config.target }} -t denoland/rusty_v8:${{ matrix.config.target }} .
```

Comment:
Where does this docker image end up? 
Maybe there is a way to put it somewhere in ./target and add the output to the github cache.
(We like to keep our CI build time reasonably short -- currently we're just under 15 minutes for most builds, ideally that wouldn't get much longer).

Reply:
To see how it fares generally, I've pushed your PR to the 'android' branch.
The first build probably will take forever. (https://github.com/denoland/rusty_v8/actions/runs/110807919)
I hope that when I run it again the cache is populated and it runs much faster.

Reply:
So a quick check [here](https://github.com/denoland/rusty_v8/runs/694004278?check_suite_focus=true#step:19:11) reveals that sccache is not being used while compiling V8... which means that every build is going to take 1.5 hours.
Is it possible to make sccache work?

Reply:
> Where does this docker image end up?
> Maybe there is a way to put it somewhere in ./target and add the output to the github cache.

I will check to add it there

> (We like to keep our CI build time reasonably short -- currently we're just under 15 minutes for most builds, ideally that wouldn't get much longer).

Totally agree

> So a quick check [here](https://github.com/denoland/rusty_v8/runs/694004278?check_suite_focus=true#step:19:11) reveals that sccache is not being used while compiling V8... which means that every build is going to take 1.5 hours.
> Is it possible to make sccache work?

I thought it was the case, I will check how we can set up that

Reply:
Ok I finally succesfully build with sccache
I couldn't use sccache server from host, even when using `--network host`  as docker option:
```
error: failed to execute compile
caused by: Failed to send data to or receive data from server
caused by: Failed to read response header
caused by: failed to fill whole buffer
```
So, I installed sccache inside containers and it worked well

Reply:
I successfully stored docker image in cache, rebuild only when Dockerfile change